packages/server/src/controllers/driver/driver-controller.ts

```diff
@@ -59,7 +60,7 @@ export const deleteDriver = async (req: Request, res: Response) => {
 	return res.send(response);
 };
 export const putDriver = async (req: Request, res: Response) => {
-	const { name, lat, lng, phone, carModel, licensePlate, date, locationId, userId } = req.body;
+	const { name, lat, lng, phone, carModel, licensePlate, date, locationName, locationId, userId } = req.body;
```

Comment:
What's the use of locationName when we can get the whole location object using API.

Reply:
Thats in order to display the name of location in the table

Reply:
Okay, but isn't the whole purpose of the locationId in the Drivers table is to cross-reference it with the Locations table and get the information we need?

Reply:
Yeah correct but when i wanted to display location's name thats retrieved from the cross-referenced information on the table, it became troublesome since it was difficult to add it to the rowData object which is displayed on each row in the table, basically what i was telling you this morning

Reply:
Can you ask Hamzah if this what we should do?

Reply:
Okay

Reply:
if we only want to display the locationName then thats ok.
if not then lets return all the related fields of the location.

Reply:
Yeah thats to display the locationName in the table